templates/bot/js/default/package.json

```diff
@@ -18,7 +18,7 @@
   "dependencies": {
     "@microsoft/adaptivecards-tools": "^1.0.0",
     "botbuilder": "^4.18.0",
-    "restify": "~8.5.1"
+    "restify": "~10.0.0"
```

Comment:
Some are using "~" and some are using "^". How about making them consistent?

Reply:
Updated.
I'm not sure why some were using `~`, but it seems safe to use `^` for all templates.